src/java.base/share/classes/jdk/internal/loader/NativeLibraries.java

```diff
@@ -384,6 +384,20 @@ boolean open() {
         }
     }
 
+    public static NativeLibrary defaultLibrary = new NativeLibraryImpl(Object.class, "<default>", true, true) {
```

Comment:
This field can be final.

Reply:
Thanks - I already made this change in the latest revision.